Detector/DetCRD/compact/CRD_common_v01/DC_Simple_v01_01.xml

```diff
@@ -78,9 +91,10 @@
 
   <readouts>
     <readout name="DriftChamberHitsCollection">
-      <segmentation type="GridDriftChamber" cell_size="10*mm" epsilon0="Epsilon" detector_length="DC_length" identifier_phi="cellID" />
+      <segmentation type="GridDriftChamber" cell_size="10*mm" epsilon0="Epsilon" detector_length="DC_length" identifier_phi="cellID" DC_inner_rbegin="DC_inner_chamber_layer_rbegin" DC_inner_rend="DC_inner_chamber_layer_rend" DC_outer_rbegin="DC_outer_chamber_layer_rbegin" DC_outer_rend="DC_outer_chamber_layer_rend" DC_inner_rmin="SDT_inner_chamber_radius_min" DC_inner_rmax="SDT_inner_chamber_radius_max" DC_outer_rmin="SDT_outer_chamber_radius_min" DC_outer_rmax="SDT_outer_chamber_radius_max" DC_inner_layer_number="SDT_inner_chamber_layer_number" DC_outer_layer_number="SDT_outer_chamber_layer_number" safe_distance="DC_safe_distance" layerID="layer" layer_width="SDT_chamber_layer_width"/>
 
-      <id>system:8,chamber:1,layer:8,cellID:16</id>
+      <!--<id>system:8,chamber:1,layer:8,cellID:16</id> -->
+      <id>system:5,side:-2,layer:9,chamber:8,sensor:8,cellID:16</id>
```

Comment:
Hi @myliu-hub ,
Could you explain these variables a bit? For example, where did you define the side and so on?
Tao

Reply:
@mirguest this is from my requirement, which let the id format similar with other silicon tracker. And then the previous tracking algorithm can read the drift chamber data smoothly. The silicon tracker id follows `<id>system:5,side:-2,layer:9,module:8,sensor:8</id>`, so I suggest to put chamber at same position with module.